driver/src/test/java/org/neo4j/driver/internal/ConfigTest.java

```diff
@@ -26,14 +26,12 @@
 
 import static org.hamcrest.CoreMatchers.equalTo;
 import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertThat;
-import static org.junit.Assert.assertTrue;
 
 public class ConfigTest
 {
-    private static final File DEFAULT_KNOWN_CERTS = new File( System.getProperty( "user.home" ),
-            "neo4j/neo4j_known_certs" );
+    private static final File DEFAULT_KNOWN_CERTS = new File( System.getProperty( "user.home" ), ".neo4j/neo4j_known_certs" );
```

Comment:
+1, but I forgot to sync with team on that and didn't update the DIP. If nobody minds, I'm +1 to changing this to `known_hosts` as per above.